kafka08/src/test/java/zipkin2/reporter/kafka08/ITKafkaSender.java

```diff
@@ -39,12 +38,11 @@
 
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static zipkin2.TestObjects.CLIENT_SPAN;
 
 public class ITKafkaSender {
-
   @Rule public KafkaJunitRule kafka = new KafkaJunitRule();
-  @Rule public ExpectedException thrown = ExpectedException.none();
```

Comment:
@anuraaga I suspect you're happy they deprecated this and so now everything uses the same exception assertions :P